src/displayapp/screens/Music.cpp

```diff
@@ -211,31 +211,27 @@ void Music::Refresh() {
       UpdateLength();
     }
   } else {
-    lv_label_set_text(txtPlayPause, Symbols::play);
+    lv_label_set_text_static(txtPlayPause, Symbols::play);
   }
 }
 
 void Music::UpdateLength() {
   if (totalLength > (99 * 60 * 60)) {
-    lv_label_set_text(txtTrackDuration, "Inf/Inf");
+    lv_label_set_text_static(txtTrackDuration, "Inf/Inf");
   } else if (totalLength > (99 * 60)) {
-    char timer[12];
-    sprintf(timer,
-            "%02d:%02d/%02d:%02d",
-            (currentLength / (60 * 60)) % 100,
-            ((currentLength % (60 * 60)) / 60) % 100,
-            (totalLength / (60 * 60)) % 100,
-            ((totalLength % (60 * 60)) / 60) % 100);
-    lv_label_set_text(txtTrackDuration, timer);
+    lv_label_set_text_fmt(txtTrackDuration,
+                          "%02d:%02d/%02d:%02d",
+                          (currentLength / (60 * 60)) % 100,
+                          ((currentLength % (60 * 60)) / 60) % 100,
+                          (totalLength / (60 * 60)) % 100,
+                          ((totalLength % (60 * 60)) / 60) % 100);
   } else {
-    char timer[12];
-    sprintf(timer,
-            "%02d:%02d/%02d:%02d",
-            (currentLength / 60) % 100,
-            (currentLength % 60) % 100,
-            (totalLength / 60) % 100,
-            (totalLength % 60) % 100);
-    lv_label_set_text(txtTrackDuration, timer);
+    lv_label_set_text_fmt(txtTrackDuration,
+                          "%02d:%02d/%02d:%02d",
```

Comment:
non-blocking, just personal preference and curiosity: I would indent the lines with just one more indent than the parent function. Do we have a style guideline to match the indent to the function length? I personally like the "one indent more" approach better, because the formatting of the arguments doesn't need to change when the function name changes. Like it is currently all the arguments would have to be touched when the function-name is changed

Reply:
> Do we have a style guideline to match the indent to the function length?

Is this something that `clang-format` is not fixing?

Reply:
Running `clang-format` will set the indentation like this, so that's the style guideline currently.

Reply:
ah ok, thanks for the info